site/src/pages/tutorials/tutorial-blog-rest-step1.mdx

```diff
@@ -0,0 +1,107 @@
+---
+menuTitle: "Create a server"
+order: 1
+type: step
+tags:
+  - server
+level: basic
+---
+
+# Creating a server
+
+As the first step of the tutorial, we create a server instance and run a dummy service to check that the server and service are launched.
+We will use Armeria's <type://ServerBuilder> for this task.
+
+<TutorialSteps current={1} />
+
+## What you need
+
+You don't need to prepare anything at this stage.
+Just check that you have prepared the [prerequisites](./tutorial-blog-rest-intro/#prerequisites).
+
+## 1. Create the main class
+
+Start by creating a main class:
```

Comment:
Actually, I went through the tutorial and removed most of them before this commit, but I think it may not be too bad to add them back. However, I avoided repeating them too much in each step.
Plus, I've changed bits and pieces to make the tone lighter than before.
Done e330b463100dd63e5f1e299ceb0e374ba68f35a1

Reply:
Thanks for the tone! 😄 Let me take a look at them soon. ❤️